support/bin/silver

```diff
@@ -31,5 +31,17 @@ fi
 # Set flags if not overriden in environment
 SVJVM_FLAGS=${SVJVM_FLAGS:-"-Xmx5000M -Xss20M"}
 
+# Save the command line arguments to pass to Silver
+SV_ARGS="$@"
```

Comment:
shellcheck complains,

```
In support/bin/silver line 35:
SV_ARGS="$@"
        ^--^ SC2124 (warning): Assigning an array to a string! Assign as array, or use * instead of @ to concatenate.

For more information:
  https://www.shellcheck.net/wiki/SC2124 -- Assigning an array to a string! A...
```